docs/user_guide/mitigation.rst

```diff
@@ -42,8 +42,9 @@ Fairlearn contains the following algorithms for mitigating unfairness:
 
 .. [*] coming soon!
 
-DP refers to demographic parity, EO to equalized odds, TPRD to true positive
-rate difference, ERR to error rate ratio, and BGL to bounded group loss. For
+DP refers to Demographic Parity, EO to Equalized Odds, TPRP to True Positive
```

Comment:
This is hard to read. Perhaps a separate table of acronym and definition?

Reply:
We have all the moments in a separate section, but the definition of fairness constraints should probably be in a a different section altogether (the fairness in ML section). I don't quite know what you find hard to read there since everything is listed right below the table.

Reply:
I think that the list of "<Acronym> refers to <Definition>" might work better as a table.